modulo1/projeto_lista_js/lista-exercicios-intro-js/exercicios.js

```diff
@@ -0,0 +1,126 @@
+// EXEMPLOS DE IMPLEMENTAÇÃO ---------------------------------------------------------------
+
+// EXERCÍCIO 0A
+function soma(num1, num2) {
+    
+  return num1 + num2
+}
+
+// EXERCÍCIO 0B
+function imprimeMensagem() {
+  // implemente sua lógica aqui
+  const mensagem = prompt('Digite uma mensagem!')
+
+  console.log(mensagem)
+}
+
+// EXERCÍCIOS PARA FAZER ------------------------------------------------------------------t
+
+// EXERCÍCIO 01
+function calculaAreaRetangulo() {
+  // implemente sua lógica aqui
+  const altura = prompt("Digite a altura do triângulo:");
+  const largura = prompt("Digite a largura do triângulo:");
+  console.log(altura * largura)
+}
+
+// EXERCÍCIO 02
+function imprimeIdade() {
+  // implemente sua lógica aqui
+  const anoAtual = prompt("Digite o ano de atual:");
+  const anoNascimento = prompt("Digite o ano de seu nascimento:");
+  console.log(anoAtual - anoNascimento)
+
+}
+
+// EXERCÍCIO 03
+function calculaIMC(peso, altura) {
+  // implemente sua lógica aqui
+  return peso / (altura * altura)
+  
+  calculaIMC(85,1.8)
+}
+
+// EXERCÍCIO 04
+function imprimeInformacoesUsuario() {
+  // implemente sua lógica aqui
+  // "Meu nome é NOME, tenho IDADE anos, e o meu email é EMAIL."
+  const nome = prompt("Digite seu nome:")
+  const idade = prompt("Informe sua idade:")
+  const email = prompt("Digite seu e-mail:")
+  console.log(`Meu nome é ${nome}, tenho ${idade} anos, e o meu email é ${email}.`)
+}
+
+// EXERCÍCIO 05
+function imprimeTresCoresFavoritas() {
+  // implemente sua lógica aqui
+  const cor1 = prompt("Digite seu cor favorita:")
+  const cor2 = prompt("Informe uma segunda cor favorita:")
+  const cor3 = prompt("Digite uma terceira cor favorita:")
+
+  const array = [cor1, cor2, cor3]
+  console.log(array)
+}
+
+// EXERCÍCIO 06
+function retornaStringEmMaiuscula(string) {
+  // implemente sua lógica aqui
+  return string.toUpperCase()
+
+}
+
+// EXERCÍCIO 07
+function calculaIngressosEspetaculo(custo, valorIngresso) {
+  // implemente sua lógica aqui
+  return custo / valorIngresso
+
+}
+
+// EXERCÍCIO 08
+function checaStringsMesmoTamanho(string1, string2) {
+  // implemente sua lógica aqui
+  return string1.length === string2.length
+}
+
+// EXERCÍCIO 09
+function retornaPrimeiroElemento(array) {
+  // implemente sua lógica aqui
+  return array[0]
+
+}
+
+// EXERCÍCIO 10
+function retornaUltimoElemento(array) {
+  // implemente sua lógica aqui
+  return array.pop()
+}
+t
+// EXERCÍCIO 11
+function trocaPrimeiroEUltimo(array) {
+  // implemente sua lógica aqui
+  
+}
+
+// EXERCÍCIO 12
+function checaIgualdadeDesconsiderandoCase(string1, string2) {
+  // implemente sua lógica aqui
+  return string1.toUpperCase === string2.toUpperCase
+}
```

Comment:
Quando for utilizar o toUpperCase lembra de colocar no final dele os parênteses, pois ele é um método. exemplo:
```
function checaIgualdadeDesconsiderandoCase(string1, string2) {
   return string1.toLowerCase() === string2.toLowerCase()
}

```